src/test/java/cloud/fogbow/ras/core/plugins/interoperability/azure/util/AzureStateMapperTest.java

```diff
@@ -0,0 +1,52 @@
+package cloud.fogbow.ras.core.plugins.interoperability.azure.util;
+
+import cloud.fogbow.ras.api.http.response.InstanceState;
+import cloud.fogbow.ras.core.models.ResourceType;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AzureStateMapperTest {
+
+    // test case: When calling the map method with compute type and creating state,
+    // it must verify if It returns the instance creating state.
+    @Test
+    public void testMapSuccessfullyWhenCreatingState() {
+        // set up
+        ResourceType resourceType = ResourceType.COMPUTE;
+
+        // exercise
+        InstanceState instanceState = AzureStateMapper.map(resourceType, AzureStateMapper.CREATING_STATE);
+
+        // verify
+        Assert.assertEquals(InstanceState.CREATING, instanceState);
+    }
+
+    // test case: When calling the map method with compute type and creating state,
+    // it must verify if It returns the instance creating state.
+    @Test
+    public void testMapSuccessfullyWhenSuccededState() {
```

Comment:
The test method name is mistyped. The state is written as **succeded** when it should be **succeeded**

Reply:
Done. 
I've made the change in the branch azure-compute-plugin-four-pr-fixes. [Commit reference](https://github.com/fogbow/resource-allocation-service/commit/96de23de99b849448e5af754d4117505f00b3e53).